lib/migrations/2020-01-15.ts

```diff
@@ -298,13 +299,18 @@ export class Migration20200115 extends Migration {
     const rawPasscodeParams = await this.services.deviceInterface.getJsonParsedStorageValue(
       LegacyKeys.MobilePasscodeParamsKey
     );
+    const rawPasscodeKeyboardType = await this.services.deviceInterface.getJsonParsedStorageValue(
+      LegacyKeys.MobilePasscodeKeyboardTypeKey
+    );
     const rawStructure: StorageValuesObject = {
       [ValueModesKeys.Nonwrapped]: {
         [StorageKey.WrappedRootKey]: wrappedAccountKey,
         [StorageKey.RootKeyWrapperKeyParams]: rawPasscodeParams,
-        [StorageKey.RootKeyParams]: rawAccountKeyParams
+        [StorageKey.RootKeyParams]: rawAccountKeyParams,
+      },
+      [ValueModesKeys.Unwrapped]: {
```

Comment:
What is the difference between `Unwrapped` and `Nonwrapped` in case of `ValueModesKeys`? Maybe it should be documented more explicitly. Because biometrics are set on `Unwrapped` and then we use `StorageValueModes.Nonwrapped` in the mobile repo to read it.

Reply:
Unwrapped means the value was wrapped previously (encrypted), then unwrapped and placed in working memory. Nonwrapped means the value should never be encrypted. So any values that are required to begin the unlocking process of the application should be nonwrapped (such as passcode keyboard type). This way we can access them before the user authenticates.